lib/generators/templates/setup/bin_setup.rb

```diff
@@ -20,11 +20,8 @@ def using_node?
   system("yarn install --check-files") if using_node?
 
   puts "\n== Preparing database and adding development seed data =="
-  if File.exist? "lib/tasks/dev.rake"
-    system! "bin/rails dev:prime"
-  else
-    system! "bin/rails db:prepare"
-  end
+  system! "bin/rails db:prepare"
+  system! "bin/rails development:seed" if File.exist? "lib/tasks/development.rake"
```

Comment:
@nickcharlton 

> One of the nice things about `dev:prime` was that it read nicely when in scripts. I'd not call any other database tasks, just that one and it looked good:
> 
> https://github.com/thoughtbot/administrate/blob/9870b40ce7330e64acad04a6c1685299defff77b/bin/setup#L25

Are you suggesting something like this?

```suggestion
  system! "bin/rails development:seed" if File.exist? "lib/tasks/development.rake"
```

Reply:
Right — you don't need to call `db:prepare`, because the task for seeding calls it in it's dependency chain.

Reply:
It does? I might be misunderstanding, but the newly introduced `development:seed` has no other dependencies other than the `:environment`. 

https://github.com/thoughtbot/suspenders/blob/04589ab9ccc8472031de30d8e0a714560d4be5ba/lib/generators/templates/tasks/development.rake#L3

Reply:
I mean that in comparison to the original (not the new one!), one of the "hidden" features of it was that it'd also setup the database for you. This is why Administrate's `bin/setup` is so small.